appveyor.yml

```diff
@@ -1,10 +1,13 @@
 version: v1.11.0.build{build}
 
+image:
```

Comment:
I agree on principle... was looking for fast feedback from AppVeyor (and from you).